doc/repos.md

```diff
@@ -48,6 +48,12 @@ $repos = $client->api('repo')->find('chess', array('language' => 'php', 'start_p
 $repo = $client->api('repo')->show('KnpLabs', 'php-github-api')
 ```
 
+or
+
+```php
```

Comment:
Maybe add the note about the undocumented feature also here in the docs. This makes it more visible for users using this api method. We can remove the note later when it's official mentioned in the docs!